types/xk6-sql-browser/tslint.json

```diff
@@ -0,0 +1,4 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+	 "npm-naming": false
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/xk6-sql-browser/tslint.json;
  Line: 3;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.